plural/helm/console/values.yaml.tpl

```diff
@@ -68,9 +68,8 @@ secrets:
   identity: {{ readFile $identity | quote }}
 {{ else if ne (dig "console" "secrets" "identity" "default" .) "default" }}
   identity: {{ .console.secrets.identity | quote }}
-{{ else }}
-  key: {{ readFile (homeDir ".plural" "key") | quote }}
 {{ end }}
+  key: {{ dedupe . "console.secrets.key" (readFile (homeDir ".plural" "key")) | quote }}
```

Comment:
Should this be deduped, or always read from the file? The previous logic would always read it from the file.

Reply:
Also, what if the `.plural/key` file doesn't exist. You'd need to do a check if that file exists first before doing this line of templating.

Reply:
there are cases where it is not actually in the file.  Also i don't think you can really run `plural build` w/ the key existing, but the function will just return an empty string if it doesn't